src/store/modules/transactions/actions.js

```diff
@@ -44,7 +48,9 @@ export default {
    * @return {Promise<*>}
    */
   async getTransactionsFromBlockHash ({ state, commit, dispatch }, hash) {
-    const client = await ae(this.state.epochUrl)
+    const client = await EpochChain({
+      url: this.state.epochUrl
+    })
```

Comment:
I think we should extract duplicate
```js
    const client = await EpochChain({
      url: this.state.epochUrl
    })
```
somehow.

Reply:
Would it be good if I take this in backlog and resolve in another PR?

Reply:
Would it be good if I take this in backlog and resolve in another PR?

Reply:
Would it be good if I take this in backlog and resolve in another PR?

Reply:
Would it be good if I take this in backlog and resolve in another PR?

Reply:
Would it be good if I take this in backlog and resolve in another PR?